css/app.css

```diff
@@ -23,19 +23,27 @@ gaia-list h3 {
   width: calc(100% - 10px);
 }
 
-gaia-list .action {
-  text-align: center;
-  text-decoration: underline;
-  background-color: var(--button-background);
-}
-
 gaia-list .action.disabled {
   pointer-events: none;
   color: var(--header-disabled-button-color);
 }
 
-gaia-list .destructive {
-  color: var(--color-destructive);
+gaia-button.top {
+  min-width: 33% !important;
```

Comment:
If I don't do that, the buttons are on two lines. They still are not aligned properly.